daemon/daemon.go

```diff
@@ -121,6 +121,9 @@ type Daemon struct {
 	pruneRunning     int32
 	hosts            map[string]bool // hosts stores the addresses the daemon is listening on
 	startupDone      chan struct{}
+
+	//Stores The LB IP address for the node on a given a network id.
+	networkToNodeLBIP map[string]net.IP
```

Comment:
I think this would be more appropriate as a new type (something like LBAttachmentStore?), that way we don't add additional methods onto `Daemon` which is already terribly bloated. The new methods would be on the new type.

Unless there's already a type on daemon that could be used?

Maybe something that wraps both this and netController?

Reply:
What package would you suggest adding LBAttachmentStore to?
We can add it to daemon, but, then we need to import daemon into backend.go and that sort of feels like it violates some abstraction.
I can put it in its own package.
I can also put it in the network package under (daemon/settings.go).  I feel like this is the most appropriate spot.
Other suggestions?

Reply:
I put it in the network package under daemon/settings.go.